.github/workflows/maven-matrix.yml

```diff
@@ -38,6 +37,35 @@ jobs:
       - name: Build and test with Maven
         run: $MVN_CMD install
 
+  build-and-test-with-es-version:
+    needs: build-and-test-with-jdk
+    strategy:
+      matrix:
+        elasticsearchVersion: [ "7.10.0", "7.9.3", "7.8.1",
+                                "7.7.1", "7.6.2", "7.5.2" ]
+      fail-fast: false
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.m2/repository
+            !~/.m2/repository/com/senacor
+            ~/.m2/wrapper
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Build and test with Maven
+        run: $MVN_CMD install -DskipTests
+      - name: elasticsearch test version ${{ matrix.elasticsearchVersion }}
+        run: $MVN_CMD --file ./tests/pom.xml clean verify -Delasticsearch.version=${{ matrix.elasticsearchVersion }}
 
   release:
     # Release to maven central and create Github release
```

Comment:
`release.needs` should require both jobs:
```
needs: [build-and-test-with-jdk, build-and-test-with-es-version]
```

Reply:
done